frontend/src/scenes/surveys/Survey.tsx

```diff
@@ -526,20 +529,28 @@ export function SurveyView({ id }: { id: string }): JSX.Element {
                                                                     <CodeSnippet language={Language.JavaScript} wrap>
                                                                         {OPT_IN_SNIPPET}
                                                                     </CodeSnippet>
-                                                                    <div className="flex gap-1 items-center">
-                                                                        <LemonCheckbox checked={!!surveyPlugin} />{' '}
-                                                                        {surveyPlugin ? (
-                                                                            <span>Install survey app</span>
-                                                                        ) : (
-                                                                            <LemonButton onClick={installSurveyPlugin}>
-                                                                                Install the survey app
-                                                                            </LemonButton>
-                                                                        )}{' '}
-                                                                        {installingPlugin && <Spinner />}
-                                                                    </div>
+                                                                    {user &&
+                                                                    canGloballyManagePlugins(user.organization) ? (
```

Comment:
@liyiy this flow overall doesn't really make sense since no one on cloud can manage plugins.

Instead, this should be a flow for enabling the plugin, rather than installing it.

Reply:
Ah I see, that's what this PR turns it into, effectively